algebra/module.lean

```diff
@@ -233,6 +233,21 @@ end
 
 end is_submodule
 
+section comm_ring
+
+theorem is_submodule.eq_univ_of_contains_unit {α : Type u} [comm_ring α] (S : set α) [is_submodule S] :
+(∃ x ∈ S, ∃ y, y * x = (1:α)) → S = set.univ :=
```

Comment:
replace the existential by `(x y : α) (hx : x ∈ S) (h : y * x = 1) : S = set.univ`